flink-libraries/flink-cep/src/test/java/org/apache/flink/cep/nfa/NFAITCase.java

```diff
@@ -272,37 +372,42 @@ public void testBranchingPattern() {
 			public boolean filter(Event value) throws Exception {
 				return value.getName().equals("start");
 			}
-		})
-			.followedBy("middle-first").subtype(SubEvent.class).where(new FilterFunction<SubEvent>() {
+		}).followedBy("middle-first").subtype(SubEvent.class)
+			.where(new FilterFunction<SubEvent>() {
 				private static final long serialVersionUID = 6215754202506583964L;
 
 				@Override
 				public boolean filter(SubEvent value) throws Exception {
-					return value.getVolume() > 5.0;
+					return value.getVolume() >
```

Comment:
this can be as before, in one line. In general we try to minimize unnecessary changes ;).